orchagent/aclorch.cpp

```diff
@@ -1060,6 +1060,108 @@ void AclOrch::doTask(Consumer &consumer)
     }
 }
 
+void AclOrch::addAclTable(AclTable &newTable, string table_id)
+{
+    sai_object_id_t table_oid = getTableById(table_id);
+
+    if (table_oid != SAI_NULL_OBJECT_ID)
+    {
+        // table already exists, delete it first
```

Comment:
This depends on what the init design is. If the design is to remove first if the table exists, then this is needed. Since I am not sure about the initial design. I would rather leave it here.

Reply:
The initial design won't work, especially if the table has rules associated with it.  This will trigger SAI level failure due to the resource busy.

Reply:
I agree the initial design will cause SAI level failure. That will require more in-depth change, which is beyond the scope of this pr. We need to change the AclOrch design.